resources/views/auth/forgot-password.blade.php

```diff
@@ -1,20 +1,20 @@
 @extends('layouts.app')
 
 @section('title')
-    @lang('fortify::metatags.reset-password')
+    {{ trans('metatags.reset-password') ?? trans('fortify::metatags.reset-password') }}
```

Comment:
it would be better to make use of slots or using `@yield` so the caller can overwrite it by using a custom slot, instead of forcing a specific translation by the package